arangod/Aql/SortNode.cpp

```diff
@@ -264,7 +264,7 @@ std::unique_ptr<ExecutionBlock> SortNode::createBlock(
   ExecutionNode const* previousNode = getFirstDependency();
   TRI_ASSERT(previousNode != nullptr);
 
-#ifdef USE_IRESEARCH
+#if 0
```

Comment:
Yes, I kept it because I had the same discusson with @gnusi a while ago, and he wanted to keep the code even though it was disabled. But maybe @gnusi should comment himself on this.

Reply:
gnusi is fine with removing it, so I will do it...